.github/workflows/build.yml

```diff
@@ -0,0 +1,130 @@
+name: Build and Release
+
+on:
+  push:
+    branches:
+      - main
+      - development
+      - rc-v*
+      - release/*
+      - hotfix/*
+    pull_request:
+      types: [ready_for_review, opened, reopened, auto_merge_enabled]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - targetPlatform: linux
+            os: ubuntu-latest
+          - targetPlatform: macos
+            os: ubuntu-latest
+          - targetPlatform: windows
+            os: windows-latest
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+
+      - name: Install
+        shell: bash
+        run: yarn
+
+      - name: Setup OS variables
+        run: bash scripts/configure-artifact.sh ${{ matrix.targetPlatform }}
+
+      - name: Copy config.json from the last release
+        shell: bash
+        run: |
+          mkdir pack-dist
+          set -ex
+          mkdir -p dist
+          curl -o "dist/config.json" 'https://release.nine-chronicles.com/9c-launcher-config.json'
```

Comment:
I think the `rc-v*` builds should get config.json from `internal/` folder

Reply:
This action simply migrates the existing CircleCI action. I will perform refactoring later.